src/main.rs

```diff
@@ -344,9 +344,12 @@ fn configure_app(app: &mut web::ServiceConfig, conf: &MiniserveConfig) {
             if show_hidden {
                 files = actix_files::Files::new(&full_route, path)
                     .show_files_listing()
+                    .disable_content_disposition()
```

Comment:
Now what this condition block is growing, how about trying to unify it a little? Maybe something like:
```rust
let files = actix_files::Files::new(&full_route, path).show_files_listing().disable_content_disposition();
if show_hidden {
    let files = files.use_hidden_files();
}
```
Would that work?